invenio_app_rdm/config.py

```diff
@@ -529,6 +529,8 @@ def files_rest_permission_factory(obj, action):
 SEARCH_UI_SEARCH_TEMPLATE = 'invenio_app_rdm/records/search.html'
 """Search page's base template."""
 
+_DASHBOARD_ROUTES = ["uploads", "communities", "requests"]
```

Comment:
just for me to understand the concept: do you want to keep the variable "local/protected" because it is too complex to be overridden?

Reply:
I guess that from Flask views point of view it is easy to add a new route but then adding a whole UI view it's more complex at the moment. That's why not exposing it as a configurable option makes probably sense.